### PR TITLE
Python polars article code

### PR DESCRIPTION
**Pull request by hfhoffman1144**
Python Polars showcase code

### CODE REVIEW

File: python-polars/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Pandas-Groupby
```
Comment (KateFinegan) [inline python-polars/README.md:1]: @hfhoffman1144 Is there a reason that Pandas-Groupby is the title here? I would suggest changing it to match the tutorial's title.

File: python-polars/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+Supporting code for the article [Python Polars: A Lightening-Fast DataFrame Library](https://realpython.com/polars-python-lightening-fast-dataframe-library/).
```
Comment (gahjelle) [inline python-polars/README.md:3]: I suggest that we add a small paragraph about necessary dependencies as well. I think the readers need to do something like `python -m pip install "polars[numpy,pandas]" requests matplotlib` in order to run the examples.

File: python-polars/downloads.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import requests
+import pathlib
+
+
+def download_file(file_url: str, local_file_path: pathlib.Path) -> None:
+    "Download a file from the URL and save it with the specified file name."
```
Comment (gahjelle) [inline python-polars/downloads.py:6]: Per convention, we should use a triple-quoted string for doc-strings:

```suggestion
    """Download a file from the URL and save it with the specified file name."""
```

File: python-polars/downloads.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import requests
+import pathlib
```
Comment (gahjelle) [inline python-polars/downloads.py:2]: Minor quibble, but we should put the standard library on top of the imports:

```suggestion
import pathlib
import requests
```

File: python-polars/scanning_data.py [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import pathlib
+from downloads import download_file
+import polars as pl
```
Comment (gahjelle) [inline python-polars/scanning_data.py:3]: Another small quibble. Let's do the local import after the third-party import:

```suggestion
import pathlib
import polars as pl
from downloads import download_file
```

**Review by gahjelle — COMMENTED** [review]
Thanks for putting together this opt-in, @hfhoffman1144 .

I only have a few quick comments that you can find in the line comments.

File: python-polars/README.md (hunk 1 above)
Comment (hfhoffman1144) [reply python-polars/README.md:1]: Sorry about that, I copied the README template from another article and forgot to change the main header 😅

**Review by gahjelle — APPROVED** [review]
Looks great! :+1:

**Pull request merged**